event-streams/src/event-streams-v1.0-wd01.md

```diff
@@ -0,0 +1,547 @@
+[`event-streams-timestamp`](#2212-event-streams-timestamp) delivery annotation
+properties are eligible to be used with this filter.
+
+The property values match the given values in the filter expression, if the value
+of the property is greater or equal to the value given in the filter expression.
+
+``` XML
+<type name="event-streams-delivery-annotations-filter" class="restricted" source="amqp:map" provides="filter">
+<descriptor name="amqp:event-streams-delivery-annotations-filter" code="0x00000000:0x00000200"/>
+</type> 
+```
+
+The filter-type is “amqp:event-streams-delivery-annotations-filter”, with
+type-code 0x00000000:0x00000200
+
+# 2.2.2.2 event-streams-sql filter 
+
+The event streams SQL filter expression is a constrained subset of the SQL filter
+expression defined in [AMQP filter expressions [FILTEX, 6]](#filtex-v10).
+
+The grammar builds on the grammar defined for those SQL filters, but is
+constrained as follows:
+
+```
+predicate ::=  <boolean_constant> |
+              | <expression> <comparison-operator> <expression>
+              | <predicate> <binary-logical-operator> <predicate>
+expression ::= <constant> | <field_value>  
+constant   ::= <string_constant> | <integer_constant>
+field_value := <field>
+field ::= <section> ’.’<field_name>
+section ::= { ‘message_annotations’ | ‘m’ }
+```
+
+This grammar subset allows for a minimal implementation that is limited to
+evaluating the delivery annotation properties defined here while being
+interoperable with the full SQL filter syntax.
+
+Examples:
+
+- Start from event at offset "a4c5" exclusively: `m.event-streams-offset > '100'`
+- Start from event at offset "a4c5" inclusively: `m.event-streams-offset >= '100'`
+- Start from event received after the timestamp: `m.event-streams-timestamp > 1585672841`
+
+
+``` XML
+ <type name="event-streams-sql-filter" class="restricted" source="sql-filter" provides="filter">
+<descriptor name="amqp:event-streams-sql-filter" code="0x00000000:0x00000201"/> 
+</type>
+```
+
+The filter-type is “amqp:event-streams-sql-filter”, with type-code 0x00000000:0x00000201
+
+
+## 2.3 Link ownership
+
+An implementation MAY constrain the number of concurrent receivers that can
+receive from a log or its partitions.
+
+In scenarios where a group of multiple concurrent event processors want to
+receive events from a partitioned event log and where they use an external
+consensus mechanism to agree on which event processor owns which partition, it
+is desireable to restrict each partition to one (the owning) receiver from the
+group and for event processors to "steal" ownership of the partition as
+assignments amongst the group change.
+
+This specification does not prescribe any algorithm or strategy for reaching
+such consensus, but only a mechanism for realizing its outcome:
+
+For an event log that is partitioned, a link MAY be associated with a partition
+and a consumer group when the respective link properties are provided during
```

Comment:
it's the first time "consumer group" is mentioned in this spec, right? should we define it somewhere?